client/blocks/reader-post-card/index.jsx

```diff
@@ -146,6 +139,7 @@ class ReaderPostCard extends Component {
 		const isDiscover = post.is_discover;
 		const title = truncate( post.title, { length: 140, separator: /,? +/ } );
 		const isReaderTagPage = currentRoute.startsWith( '/tag/' );
+		const isReaderDiscoverPage = currentRoute.startsWith( '/discover' );
```

Comment:
i think we already have a `isDiscoverStream` prop passed down from the discover controller we could alternatively use here.

Reply:
Yep, that will work.